src/lib/services/candidate-pool.service.ts

```
@@ -191,12 +191,42 @@ async function removeAllCandidatesFromPosition(positionId: string, orgId: string
     await prisma.candidatePoolEntry.deleteMany({ where: { positionId } });
 }
 
+async function getCandidatePoolEntry(id: string): Promise<CandidatePoolEntry | null> {
+    return prisma.candidatePoolEntry.findUnique({ where: { id } });

```

This service function should throw instead of returning null if the pool entry isn't found. This is just something we've been doing